service_directory/tests/test_admin.py

```diff
@@ -131,6 +131,13 @@ def test_location_coords_field_validation(self):
 
         self.assertContains(response, 'Invalid coordinates')
 
+    def test_location_coords_field_validation_when_blank(self):
+        data = self.get_post_payload_for_test_organisation()
+        data['location_coords'] = ''
+
+        response = self.client.post(self.api_url, data)
```

Comment:
Yes, it's just a negative test for the test above - I'll add the status code check